docs/content/preview/develop/quality-of-service/transaction-priority.md

```diff
@@ -157,3 +160,81 @@ yugabyte=> select * from account;
   </tr>
 
 </table>
+
+The following is another example which describes the usage of `yb_transaction_priority` based on the priority types. A transaction is a high priority one if it includes `SELECT`, `UPDATE` operations and so on.
+
+* From an active [ysqlsh](../../../admin/ysqlsh/#starting-ysqlsh) shell, create a table as follows:
+
+  ```sql
+  CREATE TABLE test_scan (i int, j int);
+  ```
+
+* Set `yb_transaction_priority_lower_bound` and `yb_transaction_priority_upper_bound` to be the same, which forces
```

Comment:
Remove this sentence, and choose an example with lower bound and upper bound to not be the same.